### PR TITLE
Fix documentation typo, `inin` → `in`

### PR DESCRIPTION
**Pull request by elebow**

### CODE REVIEW

File: lib/irb.rb [hunk 1]
```diff
@@ -207,7 +207,7 @@
 # in file <tt>~/.irb_history</tt>
 # (or, if a {configuration file}[rdoc-ref:IRB@Configuration+File]
 # is found, in file +.irb_history+
-# inin the same directory as that file).
+# in the same directory as that file).
```
Comment (st0012) [inline lib/irb.rb:210]: In addition to the typo, the entire sentence reads a bit weird 🤔 Would you mind improving it altogether?

Comment (elebow) [reply lib/irb.rb]: Pushed a larger change.

File: lib/irb.rb [hunk 2]
```diff
@@ -203,11 +203,10 @@
...
 #
-# By default, \IRB stores a history of up to 1000 input commands
-# in file <tt>~/.irb_history</tt>
-# (or, if a {configuration file}[rdoc-ref:IRB@Configuration+File]
-# is found, in file +.irb_history+
-# inin the same directory as that file).
+# By default, \IRB stores a history of up to 1000 input commands in a
+# file named <tt>.irb_history</tt>. The history file will be in the same directory
+# as the {configuration file}[rdoc-ref:IRB@Configuration+File] if one is found, or
+# in <tt>~/</tt> otherwise.
```
Comment (olleolleolle) [inline lib/irb.rb:209]: I like that it added more clarity on the  behaviour, too! 🚀

**Review by st0012 — APPROVED** [review]
This is a great improvement. Thank you very much 👍

**Pull request merged**